btree/btree.c

```diff
@@ -2355,6 +2355,40 @@ static void bnode_op_fini(struct node_op *op)
  *  --------------------------------------------
  */
 
+/**
+ * @brief Proposed design for using btree node to hold embedded records with
+ * indirect addressing.
+ *
+ * Fixed format node = {
+ * 	header
+ * 	dir   = {
+ * 			ok1, ok2, ok3,..., okn,
```

Comment:
Keep ok1, ov1 together also ok2,ov2 together and so on.

Reply:
Done.